src/features/CareViewSummary/components/CareViewSummary.jsx

```diff
@@ -45,6 +45,13 @@ export const CareViewSummary = (props) => {
     provider,
     refreshSummary,
   } = useContext(CareViewContext);
+
+  let localSelectedWard;
+  if (localStorage.getItem("selected_ward")) {
+    const { label } = JSON.parse(localStorage.getItem("selected_ward"));
+    localSelectedWard = label;
+  }
```

Comment:
Here we are calling the same localStorage.getItem("selected_ward") method twice.
We can do:
`const localSelectedWard = JSON.parse(localStorage.getItem("selected_ward"))?.label;`